helm/templates/_helpers.tpl

```diff
@@ -0,0 +1,80 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "amazon-cloudwatch-observability.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Name for cloudwatch-agent
+*/}}
+{{- define "cloudwatch-agent.name" -}}
+{{- default "cloudwatch-agent" .Values.agent.name }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "amazon-cloudwatch-observability.labels" -}}
+{{ include "amazon-cloudwatch-observability.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: EKS
```

Comment:
Is the EKS value constant? Customers can install on native k8s, will it also have this label?

Reply:
Yeah we will update this before we release a public helm chart. For now, the helm chart is specific for the EKS add-on.